dspace-server-webapp/src/test/java/org/dspace/app/rest/BitstreamRestControllerIT.java

```diff
@@ -51,42 +70,112 @@
  *
  * @author Tom Desair (tom dot desair at atmire dot com)
  * @author Frederic Van Reet (frederic dot vanreet at atmire dot com)
  */
-public class BitstreamContentRestControllerIT extends AbstractControllerIntegrationTest {
+public class BitstreamRestControllerIT extends AbstractControllerIntegrationTest {
 
     protected SolrLoggerService solrLoggerService = StatisticsServiceFactory.getInstance().getSolrLoggerService();
 
-    private static final Logger log = LogManager
-        .getLogger(BitstreamContentRestControllerIT.class);
-
     @Autowired
     private ConfigurationService configurationService;
 
     @Autowired
     private CitationDocumentServiceImpl citationDocumentService;
 
+    @Autowired
+    private BitstreamService bitstreamService;
+
+    @Autowired
+    private BitstreamFormatService bitstreamFormatService;
+
+    @Autowired
+    private ResourcePolicyService resourcePolicyService;
+
+    private Bitstream bitstream;
+    private BitstreamFormat supportedFormat;
+    private BitstreamFormat knownFormat;
+    private BitstreamFormat unknownFormat;
+    private ResourcePolicy resourcePolicy;
+
     @BeforeClass
     public static void clearStatistics() throws Exception {
         // To ensure these tests start "fresh", clear out any existing statistics data.
         // NOTE: this is committed immediately in removeIndex()
-        StatisticsServiceFactory.getInstance().getSolrLoggerService().removeIndex("*:*");
+//        StatisticsServiceFactory.getInstance().getSolrLoggerService().removeIndex("*:*");
     }
 
     @Before
-    public void setup() throws Exception {
+    @Override
+    public void setUp() throws Exception {
+
         super.setUp();
 
         configurationService.setProperty("citation-page.enable_globally", false);
+
+        context.turnOffAuthorisationSystem();
+
+        bitstream = bitstreamService.create(context, new StringInputStream("test"));
+
+        unknownFormat = bitstreamFormatService.create(context);
+        unknownFormat.setMIMEType("unknown test mime type");
+        unknownFormat.setDescription("unknown test description");
+        unknownFormat.setShortDescription(context, "unknown test short description");
+        unknownFormat.setSupportLevel(UNKNOWN);
+
+        knownFormat = bitstreamFormatService.create(context);
+        knownFormat.setMIMEType("known test mime type");
+        knownFormat.setDescription("known test description");
+        knownFormat.setShortDescription(context, "known test short description");
+        knownFormat.setSupportLevel(KNOWN);
+
+        supportedFormat = bitstreamFormatService.create(context);
+        supportedFormat.setMIMEType("supported mime type");
+        supportedFormat.setDescription("supported description");
+        supportedFormat.setShortDescription(context, "supported short description");
+        supportedFormat.setSupportLevel(SUPPORTED);
+
+        bitstream.setFormat(context, supportedFormat);
+
+        context.commit();
```

Comment:
If you use the `Builder` framework, this commit() should not be needed.  However, you should make sure to turn back **on** authorization (so that your tests are not running with authorization ignored).  So, this line should be `context.restoreAuthSystemState();`